Formula/qt5.rb

```diff
@@ -59,9 +68,6 @@ class Qt5 < Formula
   deprecated_option "qtdbus" => "with-dbus"
   deprecated_option "with-d-bus" => "with-dbus"
 
-  # OS X 10.7 Lion is still supported in Qt 5.5, but is no longer a reference
-  # configuration and thus untested in practice. Builds on OS X 10.7 have been
-  # reported to fail: <https://github.com/Homebrew/homebrew/issues/45284>.
   depends_on :macos => :mountain_lion
```

Comment:
Let's bump this to `:mavericks` given what we've learned from the QtWebEngine issue.